pypeit/spectrographs/spectrograph.py

```diff
@@ -796,7 +796,10 @@ def get_headarr(self, inp, strict=True):
                     return ['None']*999 # self.numhead
         else:
             hdu = inp
-        return [hdu[k].header for k in range(len(hdu))]
+        try:
+            return [hdu[k].header for k in range(len(hdu))]
+        except:
+            embed(header='802 of spectrograph')
```

Comment:
If we keep this, I've found it useful to print the exception, so:
```
except Exception as e:
    print(e)
    embed(header='802 of spectrograph')
```

Reply:
removed!